RefugeeLand.Core.Api/Brokers/Storages/StorageBroker.Refugees.cs

```diff
@@ -20,20 +20,24 @@ public async ValueTask<Refugee> InsertRefugeeAsync(Refugee refugee)
         {
             using var broker = new StorageBroker(configuration);
 
-            EntityEntry<Refugee> refugeeEntityEntry = await broker.AddAsync(refugee);
+            EntityEntry<Refugee> refugeeEntityEntry =
+                await broker.AddAsync(refugee);
 
             await broker.SaveChangesAsync();
 
             return refugeeEntityEntry.Entity;
         }
 
         public IQueryable<Refugee> SelectAllRefugees()
-            => this.Refugees;
+        {
+            using var broker = new StorageBroker(this.configuration);
+
+            return broker.Refugees;
+        }
 
         public async ValueTask<Refugee> SelectRefugeeByIdAsync(Guid refugeeId)
         {
             using var broker = new StorageBroker(configuration);
-            broker.ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
 
```

Comment:
@ElbekDeveloper, 
- Why did you choose to remove the NoTracking behavior? 
- Did you explaint it during the session? Has the session been recorded?